### DISCUSSION TITLE
给项目起一个中文名

### DISCUSSION THREAD

**Discussion by Littlefean** [graphif/project-graph#75] [功能请求]
此issue特意供讨论

**Comment by Littlefean** [comment]
小头脑风暴一下：
节点图、数据结构、神经网络
贝塞尔曲线 、神学、神圣感、哲学、普尼
快速、庞大、逻辑体系、灵感、逻辑架构者
流程图、项目管理、发展与未来、直觉、规划图

**Comment by Littlefean** [comment]
节点风暴！

**Comment by 123hi123** [comment]
我都叫這個快速圖哈哈哈

**Comment by zerobiubiu** [comment]
轻节点 或者 轻思维

**Comment by RayLam2022** [comment]
图灵

**Comment by KaiSanPao** [comment]
快项图

**Comment by Z-D-Aurelius** [comment]
建议叫它“计划-投射”，理由如下：

1.“计划”是“Project”的意义，同时也是此软件重要的和根本的功能；
2.“投射”已经成为现当代哲学中很重要的概念，同时也是“Project”一词的根本含义（pro- 向前，ject 投掷）；
3.“计划”让人联想到体系、规划与发展，让人联想到成体系的规模和效率；
4.“投射”在此也可以意指“把头脑中的理念投射到载体上”，同时让人联想到有关图形的“射影”等等，就符合了“Project-Graph”的意味；
5.“计划”带来一种完美规划、有序运转的秩序与机械之感；
6.“投射”则带来灵动和活泼之感。

综上所述，我认为“计划-投射”是一个合理的名称。

**Reply by Littlefean** [reply]
很不错，就这个了

**Comment by MeizhouWan** [comment]
图计
